src/libs/actions/Task.ts

```diff
@@ -1163,6 +1163,7 @@ export {
     clearTaskErrors,
     canModifyTask,
     setNewOptimisticAssignee,
+    getReport,
```

Comment:
Can we move this to ReportUtils as it fits there more?

Reply:
I updated